packages/fxa-settings/src/components/Settings/DataCollection/en.ftl

```diff
@@ -2,8 +2,12 @@
 
 dc-heading = Data Collection and Use
 dc-subheader = Help improve { -product-firefox-accounts }
+dc-subheader-2 = Help improve { -product-mozilla-accounts }
 dc-subheader-content = Allow { -product-firefox-accounts } to send technical and interaction data to { -brand-mozilla }.
+dc-subheader-content-2 = Allow { -product-mozilla-accounts } to send technical and interaction data to { -brand-mozilla }.
 dc-opt-out-success = Opt out successful. { -product-firefox-accounts } won’t send technical or interaction data to { -brand-mozilla }.
+dc-opt-out-success-2 = Opt out successful. { -product-mozilla-accounts } won’t send technical or interaction data to { -brand-mozilla }.
```

Comment:
Should this be using singular or plural? (Since plural is supposed to used when referring to something affecting all Mozilla accounts, not just the individual's account.)

Reply:
@bcolsson The feedback I received here was, "Those two [this string] seem to be talking about us, the team, so that makes sense for it to be plural."

Should the comment be updated to reflect this?